INSTALL.md

```diff
@@ -43,6 +43,8 @@ CONFIG_HAVE_BPF_JIT=y
 CONFIG_HAVE_EBPF_JIT=y
 # [optional, for kprobes]
 CONFIG_BPF_EVENTS=y
+# Need kernel headers through /sys/kern/kheaders.tar.xz
```

Comment:
There is typo here. It should be /sys/kernel/kheaders.tar.xz.

Reply:
Fixed.